test/parallel/test-https-client-get-url.js

```diff
@@ -46,7 +46,7 @@ const server = https.createServer(options, common.mustCall((req, res) => {
 }, 3));
 
 server.listen(0, common.mustCall(() => {
-  const u = `https://${common.localhostIPv4}:${server.address().port}/foo?bar`;
+  const u = `https://${common.localhostIP}:${server.address().port}/foo?bar`;
```

Comment:
Here we have the problem with a url that needs to be different for IPv6 vs. IPv4.

```suggestion
  const u = (common.hasIPv6 ?
                    `https://[${common.localhostIP}]:${server.address().port}/foo?bar` :
                    `https://${common.localhostIP}:${server.address().port}/foo?bar`);
```